core/vm/interpreter.go

```diff
@@ -34,6 +39,10 @@ type Config struct {
 	JumpTable *JumpTable // EVM instruction table, automatically populated if unset
 
 	ExtraEips []int // Additional EIPS that are to be enabled
+
+	// <specular modification>
+	SpecularEVMPreTransferHook EVMHook
```

Comment:
how does this end up getting set btw?

Reply:
currently this is being set [here](https://github.com/SpecularL2/specular/blob/7b6eb40e3216c164f59f866d673cff309ed37312/clients/geth/specular/cmd/geth/config.go#L201)